src/components/victory-bar/helper-methods.js

```diff
@@ -111,7 +111,10 @@ export default {
       );
 
       const text = this.getLabel(modifiedProps, datum, index);
-      const labelStyle = this.getLabelStyle(style.labels, datum);
+      const labelPadding = style.labels.padding;
+      const labelStyleObj = datum.y < 0 ? Object.assign({}, style.labels,
+        {padding: labelPadding - 7}) : style.labels;
```

Comment:
Magic number 7? Can you check that this is correct with different font sized and multi-line labels.